test/CMakeLists.txt

```
@@ -182,6 +182,16 @@ if(HAVE_LIBUDEV_H)
     add_definitions(-DHAVE_LIBUDEV_H)
 endif()
 
+set(FFmpeg_FIND_COMPONENTS AVCODEC AVFORMAT AVUTIL SWSCALE)
+include("${SDL3_SOURCE_DIR}/cmake/FindFFmpeg.cmake")
+if(FFMPEG_FOUND)
+    add_sdl_test_executable(testspriteffmpeg NO_C90 SOURCES testspriteffmpeg.c)
```

This makes sure `icon.h` is rebuilt if it becomes outdated and only `testspriteffmpeg` is built (unlikely, but possible).
```suggestion
    add_sdl_test_executable(testspriteffmpeg NO_C90 SOURCES testspriteffmpeg.c ${icon_bmp_header})
```